pro/src/pages/IndividualOfferWizard/Stocks/Stocks.module.scss

```
@@ -0,0 +1,7 @@
+@use "styles/mixins/_rem.scss" as rem;
+@use "styles/variables/_colors.scss" as colors;
+@use "styles/mixins/_fonts.scss" as fonts;
+
+.desk-callout {
+  margin-bottom: rem.torem(16px);
+  }
```

L'indentation n'est pas bonne, je pense que stylelint va planter quelque part.

---

Amine, certaines nouveau callout, comme ici, se collent en bas ou en haut aux textes autour. Avec Marie et Martin on se disait que on fait comme ça, en passant par module.scss. Sinon tout les autres callout vont être aussi décalés sans le besoin. Pas possible de généraliser indiquant directement dans callout.

---

Hello, je parlais uniquement de l'indentation du texte. Quelque chose comme ça devrait corriger le problème.

```suggestion
}
```

---

c'est fait, merci.